src/Build.UnitTests/Graph/ProjectGraph_Tests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Evaluation;
+using Xunit;
+
+namespace Microsoft.Build.Graph.UnitTests
+{
+    public class ProjectGraphTests
+    {
+        [Fact]
+        public void TestGraphWithSingleNode()
+        {
+            string projectContents = @"
+                <Project xmlns='http://schemas.microsoft.com/developer/msbuild/2003'>
+                  <PropertyGroup>
+                    <TestProperty>value</TestProperty>
+                  </PropertyGroup>
+
+                  <Target Name='Build'>
+                    <Message Text='Building test project'/>
+                  </Target>
+                </Project>
+                ";
+            string testDirectory = Path.GetTempPath();
+            string projectPath = Path.Combine(testDirectory, "ProjectGraphTest.proj");
+            File.WriteAllText(projectPath, projectContents);
+            var projectGraph = new ProjectGraph(projectPath);
+            
+            Assert.True(projectGraph.ProjectNodes.Count == 1);
```

Comment:
You should use `Assert.Equals` instead for better error messaging. When it fails you'll see something like "Expected 1 but got 2" instead of "Expected true to be false"